application/views/template_view.py

```diff
@@ -0,0 +1,17 @@
+from flask import Blueprint, make_response, request, jsonify
+from application.controllers.template_controller import TemplateController
+
+template_view = Blueprint('template_view', __name__)
+
+
+@template_view.route('/upload/<template_id>', methods=['POST'])
```

Comment:
It's worth looking at rest plus for our APIs. The main benefits I can see:
- swagger docs (http://flask-restplus.readthedocs.io/en/stable/swagger.html)
- validation (http://flask-restful.readthedocs.io/en/0.3.5/reqparse.html) We will have a more consistent pattern around error messages from APIs. It has some inbuilt validation for data structures and request formats.